Resources/doc/columns.md

```diff
@@ -432,21 +432,21 @@ SgDatatablesBundle:render:action.html.twig
 
 ### Action options
 
-| Option              | Type                   | Default | Required | Description |
-|---------------------|------------------------|---------|----------|-------------|
-| route               | null or string         | null    |          | The name of the Action route. |
-| route_parameters    | null, array or Closure | null    |          | The route parameters. |
-| icon                | null or string         | null    |          | An icon for the Action. |
-| label               | null or string         | null    |          | A label for the Action. |
-| confirm             | bool                   | false   |          | Show confirm message if true. |
-| confirm_message     | null or string         | null    |          | The confirm message. |
-| attributes          | null or array          | null    |          | HTML Tag attributes (except 'href' and 'value'). |
-| button              | bool                   | false   |          | Render a button instead of a link. |
-| button_value        | null or string         | null    |          | The button value. |
-| button_value_prefix | bool                   | false   |          | Use the Datatable-Name as prefix for the button value. |
-| render_if           | null or Closure        | null    |          | Render an Action only if conditions are TRUE. |
-| start_html          | null or string         | null    |          | HTML code before the <a> Tag. |
-| end_html            | null or string         | null    |          | HTML code after the <a> Tag. |
+| Option              | Type                     | Default | Required | Description |
```

Comment:
You have to remove extra spaces at the end of the column since you have fixed the `null or array or Closure`, they are not longer required